qemu/tests/cfg/nic_hotplug.cfg

```diff
@@ -94,3 +94,17 @@
         - used_netdev:
             used_sameid = yes
             devadd_match_string = "it's in use"
+        - with_iommu:
+            x86_64, i386:
+            only virtio_net
+            only nic_virtio
+            netdev_extra_params_hotplug_nic1 = ",vhost=on,vhostforce=on"
+            nic_extra_params_hotplug_nic1 = "disable-legacy=on,disable-modern=off,iommu_platform=on,ats=on"
```

Comment:
1. We can use `virtio_dev_xxx` to control these options.
2. `disable-legacy=on,disable-modern=off` are not required for pcie bus(q35). You can have a try.

Reply:
Hi @PaulYuuu  I understand what you mean, but the purpose of this case is to have a hotplug/unplug network card with iommu configuration. This comes from a bug. If you follow the usage provided by you, these configurations will be added to the qemu command line instead of using it for hotplug/unplug

Reply:
Oh, I see. Since those options are invalid for migration, let's keep the current usage.